science/fldigi/Portfile

```diff
@@ -18,9 +18,9 @@ long_description    Fldigi (Fast Light Digital Modem Application), is a \
 homepage            http://www.w1hkj.com
 master_sites        http://www.w1hkj.com/files/${name}/
 
-checksums           rmd160  394d3eda37f6ff7f2c12a668a667672f7b6c942d \
-    sha256  dc4864b6e63c688b9dd932587bdaf7becfa17558380f7deff1236eb6b66921f0 \
-    size    4632923
+checksums           rmd160  97319d0547cb52065d708da421e0d1b11f494ed0 \
+    sha256  004b54fa4ff03c85e088bf1706270db5f3976afcaaf1003b4d6d3b0ef8007349 \
+    size    4633597
```

Comment:
Small cosmetic request, please clean up the indentation of the checksums, so the all line up with the `rmd160` one.

Reply:
I used to set it manually before but in effect it doesn't respect the header configuration for emacs/vim and my emacs, as expected, conform to that.

Reply:
yes, sometimes you have to manually add some spaces to make things line up...

Reply:
In reality, I need to remove the header configuration and then change the alignment or use another editor. Ok I fix.

Reply:
done